bedrock/bedrock-common/src/main/java/com/nukkitx/protocol/bedrock/data/inventory/Container.java

```diff
@@ -0,0 +1,64 @@
+package com.nukkitx.protocol.bedrock.data.inventory;
+
+public enum Container {
```

Comment:
By the looks of the values, I think this should instead be called `ContainerSlotType`